controllers/api_v1/token.go

```diff
@@ -0,0 +1,23 @@
+package api_v1
+
+import (
+	"net/http"
+
+	"github.com/gutakk/go-google-scraper/oauth"
+
+	"github.com/gin-gonic/gin"
+)
+
+type TokenAPIController struct{}
+
+// TODO: Unit test in login API PR as grant type need to change to password
+func (t *TokenAPIController) ApplyRoutes(engine *gin.RouterGroup) {
+	engine.POST("/token", t.generateToken)
+}
+
+func (t *TokenAPIController) generateToken(c *gin.Context) {
```

Comment:
While creating a token is a small piece of code and so seems it could be added to the controller, but this is not consistent with `OAuthController` which uses the OAuth Service package. Also, this controller now has a hard dependency on the package `oauth`. As discussed before, only exposes services to the controllers for better architecture patterns. 
So I recommend moving the `generateToken` method to the OAuth Service package.

Reply:
Fixed in b09ee8387fbed21b7a76ec97738e0d11915e3794